securesystemslib/serialization.py

```diff
@@ -30,26 +30,22 @@ def deserialize(self, raw_data: bytes, cls: Any) -> Any:
 class JSONDeserializer(BaseDeserializer):
     """Provides raw to JSON deserialize method."""
 
-    def deserialize(self, raw_data: bytes, cls: "JSONSerializable") -> Any:
-        """Deserialize utf-8 encoded JSON bytes into an instance of cls.
+    def deserialize(self, raw_data: bytes) -> Any:
```

Comment:
```suggestion
    def deserialize(self, raw_data: bytes) -> Dict:
```
needs to be imported from typing